eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -522,21 +602,23 @@ function FindOrCreatePackageGroupParent($serviceName, $packageDisplayName, $outp
   return $workItem
 }
 
-function FindOrCreateServiceParent($serviceName, $outputCommand = $true)
+function FindOrCreateServiceParent($serviceName, $outputCommand = $true, $ignoreReleasePlannerTests = $true)
 {
-  $serviceParent = FindParentWorkItem $serviceName -outputCommand $outputCommand
+  $serviceParent = FindParentWorkItem $serviceName -packageDisplayName $null -outputCommand $outputCommand -ignoreReleasePlannerTests $ignoreReleasePlannerTests
   if ($serviceParent) {
+    Write-Host "Found existing service work item [$($serviceParent.id)]"
     return $serviceParent
   }
 
   $fields = @()
   $fields += "`"PackageDisplayName=`""
   $fields += "`"ServiceName=${serviceName}`""
+  $fields += "`"Custom.EpicType=Service`""
```

Comment:
Do we need to care about Sub-Service type or Component type?

Reply:
We create another Epic Item as child of Service Epic for the product. I am assuming that's what used currently for subservice name as well.

Reply:
We are managing subservice as another EPIC. Same as previously done before

Reply:
It is fine if it is another Epic but does this automation need to handle those as well?

Reply:
Prepare release scrip is already handling it by creating a child under Service WI and then creating individual language package items under this subservice/product WI.